docs/source/getting-started/authenticators-users-basics.md

```diff
@@ -90,7 +90,7 @@ file is:
 c.LocalAuthenticator.create_system_users = True
 ```
 
-Adding a user to the Hub that doesn't already exist on the system will
+Adding a user to the Hub that doesn't exist already on the system will
```

Comment:
I think `already exist` is more natural here.

Reply:
Ok, I understand. I will make the necessary changes.